### ISSUE TITLE
Commander core:status command lists some options but does not support them

### ISSUE THREAD

**Issue by fchavant** [LiskHQ/lisk-sdk#3906]
### Expected behavior

`core:status` options work as documented:

```
lisk core:status --help
Show the status of a Lisk Core instances.

USAGE
  $ lisk core:status [NAME]

ARGUMENTS
  NAME  Lisk Core installation directory name.

OPTIONS
  -j, --[no-]json  Prints output in JSON format. You can change the default behaviour in your config.json file.
  --[no-]pretty    Prints JSON in pretty format rather than condensed. Has no effect if the output is set to table. You can change the default behaviour in your config.json file.

EXAMPLES
  core:status
  core:status mainnet-latest
```

For instance `lisk core:status --no-json` should show the status of a Lisk Core instances (sic) in plain text.

### Actual behavior

None of the options work; they even get treated as arguments(?!)

```
lisk core:status --json
Lisk Core instance: --json doesn't exists, Please install using lisk core:install
lisk core:status --json
Lisk Core instance: --json doesn't exists, Please install using lisk core:install
lisk core:status --pretty
Lisk Core instance: --pretty doesn't exists, Please install using lisk core:install
lisk core:status --no-pretty
Lisk Core instance: --no-pretty doesn't exists, Please install using lisk core:install
lisk core:status --no-pretty lisk-devnet
 ›   Error: Unexpected argument: lisk-devnet
 ›   See more help with --help
```

### Steps to reproduce

Run e.g `lisk core:status --json`.

### Which version(s) does this affect? (Environment, OS, etc...)
`2.2.0`

**Comment by ManuGowda** [comment]
This issue is addressed as part of #4084 , and the decision was taken to not provide `--json` and `--pretty`. However, if a user wants to change the `json` or `pretty` format, they can use `lisk config:set json true` for `json` and `lisk config:set json false` for pretty print.

**Comment by fchavant** [comment]
> However, if a user wants to change the json or pretty format, they can use lisk config:set json true for json and lisk config:set json false for pretty print.

This makes commander hardly usable in scripts.
